### PR TITLE
Added requested fields to facilities and studios, updated schemas - closes #27

### LINKED ISSUE #27: Discussion of expansion on fields required.
I and others have suggested expanding on the fields against each facility.

I would suggest we have a discussion on this topic in general.  Should we add fields or just keep it simply? 
If we decide to add fields, what fields and the justification for them.

i.e.
url: pointing at company website
Transalation additions
Date changed

As some examples.
Is this a topic for the next meeting?

### PR DESCRIPTION
**Pull request by SteveLLamb**
Updated the schema to allow for new fields on both Facilities and Studios.
- Added URLs for recently added entires for both registries (didn't add contact info - they didn't have the option to opt out at the time of info added via the google forms, 3 of 5 are EU companies)
- Added URLs for Deluxe facilities, and contact info for Deluxe Burbank as an example.

### CODE REVIEW

File: src/main/data/facilities.json [hunk 1]
```diff
@@ -387,7 +387,7 @@
   },
   {
     "code": "BKH",
-    "description": "Blockhead VFX"
+    "description": "Blockhead VFX"
```
Comment (radford-for-smpte) [inline src/main/data/facilities.json:390]: Can we exclude this whitespace hunk?

File: src/main/data/facilities.json [hunk 2]
```diff
@@ -2704,7 +2718,8 @@
   },
   {
     "code": "OND",
-    "description": "old/new cineproductions"
+    "description": "old/new cineproductions",
+    "url": "www.oldnew.de"
```
Comment (radford-for-smpte) [inline src/main/data/facilities.json:2722]: Should we always have a scheme? I think this should be:

> https://oldnew.de

File: src/main/schemas/facilities.schema.json [hunk 3]
```diff
@@ -7,6 +7,25 @@
     "code" : {
         "type": "string",
         "pattern": "^[A-Z0-9]{2,3}$"
+    },
+    "url": {
+      "type": "string"
```
Comment (radford-for-smpte) [inline src/main/schemas/facilities.schema.json:12]: Can we constrain this using `format: "uri"`?

File: src/main/schemas/facilities.schema.json [hunk 4]
```diff
@@ -7,6 +7,25 @@
     "code" : {
         "type": "string",
         "pattern": "^[A-Z0-9]{2,3}$"
+    },
```
Comment (radford-for-smpte) [inline src/main/schemas/facilities.schema.json:10]: I think the schema version should get updated in connection w/ this change.

File: src/main/data/facilities.json (hunk 1 above)
Comment (SteveLLamb) [reply src/main/data/facilities.json:390]: Yep. Will update.

File: src/main/data/facilities.json (hunk 2 above)
Comment (SteveLLamb) [reply src/main/data/facilities.json:2722]: I left this for ideas for validation, as noted next comment.

File: src/main/schemas/facilities.schema.json (hunk 3 above)
Comment (SteveLLamb) [reply src/main/schemas/facilities.schema.json:12]: I tried that, but it doesn't seem to work correctly, and any value seems to pass validation. I didn't want to validation via regex if at all possible, too many variables. Ideas?

File: src/main/schemas/facilities.schema.json (hunk 4 above)
Comment (SteveLLamb) [reply src/main/schemas/facilities.schema.json:10]: Roger that, will update.

**Comment by SteveLLamb** [comment]
> Any interest in also updating the test suites in in `test/`?

Yea sure, I didn't see those.

File: src/main/data/facilities.json (hunk 2 above)
Comment (SteveLLamb) [reply src/main/data/facilities.json:2722]: http:// or https:// validates on the uri type, so yes.

File: src/main/schemas/facilities.schema.json (hunk 3 above)
Comment (SteveLLamb) [reply src/main/schemas/facilities.schema.json:12]: I got the URI format working on the schema. Also swapped the regex on email for format email, which seems to work pretty well.

Comment (radford-for-smpte) [reply src/main/schemas/facilities.schema.json]: This seems to work great:

```
"url": {
      "type": "string",
      "format": "uri",
      "pattern": "^http(s)?"
 },
```

Comment (SteveLLamb) [reply src/main/schemas/facilities.schema.json]: Just tested, agreed. Will add in the pattern as well.

**Comment by SteveLLamb** [comment]
> Any interest in also updating the test suites in in `test/`?

@radford-for-smpte These have been updated.

**Comment by SteveLLamb** [comment]
@radford-for-smpte I believe all requested changes and updates have been made. Can you please re-review so I can work on getting the fields added to the registries-site pages?

**Pull request merged**